composer.json

```diff
@@ -14,13 +14,13 @@
     ],
     "license": "BSD-3-Clause",
     "require": {
-        "silverstripe/recipe-cms": "^4@dev",
-        "silverstripe/vendor-plugin": "^1@dev",
-        "dnadesign/silverstripe-elemental": "^4@dev",
-        "sheadawson/silverstripe-linkable": "^2@dev"
+        "silverstripe/recipe-cms": "^4.0",
```

Comment:
we should be able to remove recipe-cms, and likely vendor-plugin

Reply:
nice catch! updated